qds_sdk/qubole.py

```diff
@@ -87,3 +89,8 @@ def agent(cls, version=None):
           cls.cached_agent = Connection(cls._auth, cls.rest_url, cls.skip_ssl_cert_check)
 
         return cls.cached_agent
+
+    @classmethod
+    def get_cloud(cls):
+        return Connection(cls._auth, cls.baseurl.rstrip("/").replace(cls.baseurl.rstrip("/").split("/")[-1], "")
```

Comment:
Cached connection always have base url like https://api.qubole.com/api/v1.2.Thats why this change was done.  I have already merged changes of about api call to master. Will add it once its deployed on environment so that I can test it.
https://bitbucket.org/qubole/qbol/commits/8216e4a939672f5bf1187e0087e5850363322a09?at=master